knative-operator/cmd/manager/main.go

```diff
@@ -97,11 +90,21 @@ func main() {
 		os.Exit(1)
 	}
 
-	// Setup all webhooks
-	if err := webhook.AddToManager(mgr); err != nil {
-		log.Error(err, "")
-		os.Exit(1)
-	}
+	// Setup all Webhooks
+	hookServer := mgr.GetWebhookServer()
+	hookServer.Port = 9876
+	hookServer.CertDir = "/apiserver.local.config/certificates"
+	hookServer.KeyName = "apiserver.key"
+	hookServer.CertName = "apiserver.crt"
```

Comment:
Do we care about `hookServer.ClientCAName` in any case for api server validation?

Reply:
Nope, don't think so.